packages/fast-components-react-base/src/image/README.md

```diff
@@ -1,8 +1,5 @@
 # Image
-Use *image* to break up text-heavy pages, to illustrate products for sale, or to improve storytelling on a page. The use of *image* should always support the page’s story and, when possible, contribute to that story. Size the *image* to complement the rest of the page content. Use the **picture** option to employ a selection of image versions for different viewport sizes.
-
-## Style guidance
-Don't use *image* as a substitute for *hero*, *feature*, or *product* and *content placements*. Don't use an image if it reduces the effectiveness of the page.
+Use *image*, based on the standard [image](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) and [picture](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture) elements, to break up text-heavy pages, to illustrate products for sale, or to improve storytelling on a page. Use the **picture** option when using a selection of image versions for different viewport sizes.
```

Comment:
Can we find a way to remove this:

"to break up text-heavy pages, to illustrate products for sale, or to improve storytelling on a page. "

That seems very "brand" guidance specific and not necessarily helpful for the base component.

Reply:
Sure will remove.